src/plugins/intel_gpu/src/kernel_selector/jitter.cpp

```diff
@@ -1799,6 +1799,16 @@ JitConstants FusedOpsCodeGenerator::MakeOpJitConstants(const FusedOpsConfigurati
 
             auto tmp_var = out_var + "_tmp";
             op_decls += "\\\n\t" + GetType(get_acc_t(), vec_size) + " " + tmp_var + " = " + input_vars[0] + op + input_vars[1] + ";";
+
+            // Add round for float-to-int case to avoid 'convert_int(7.999999523163) returns 7'
+            auto in_type = get_acc_t();
+            auto out_type = desc.output_tensor.GetDType();
+            if ((in_type == Datatype::F16 || in_type == Datatype::F32) &&
+                (out_type != Datatype::BINARY && out_type != Datatype::F16 && out_type != Datatype::F32) &&
+                vec_size == 1) {
+                tmp_var = "round(" + tmp_var + ")";
+            }
```

Comment:
As you know, it also converts 7.6 into 8. I think it can make some accuracy issues.

Reply:
I've changed to add m_pythondiv param as spec doc for eltwise instead of adding "round" operation at cl.